### PR TITLE
Update travis

### PR DESCRIPTION
**Pull request by amitkumarj441**
### - What I did
         Added installation script.

### CODE REVIEW

**Comment by DavidKnott** [comment]
Why is this necessary?  It's my understanding that everything that needs to be installed gets installed in `install:`

**Comment by amitkumarj441** [comment]
Actually separate `setup.py` runs on travis to lower memory usage and I think travis needs a fix by adding `dev_requirements.txt`

**Comment by DavidKnott** [comment]
@amitkumarj441 So by including `setup.py` in the scrips it prevents travis from doing it in the background and this ends up using less memory?

File: .travis.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ install:
   - pip install .
 
 script:
+  - python setup.py install
```
Comment (jacqueswww) [inline .travis.yml]: There is no point in this because we already do the `pip install .` as part of the install section.

**Comment by jacqueswww** [comment]
Closing as it seems all our CI is still working fine.

**Pull request closed without merging**